### DISCUSSION TITLE
Is it possible to run a previous version (v 1) until the new release is stable?

### DISCUSSION THREAD

**Discussion by maiya-22** [dendronhq/dendron#2147] [Q&A]
Hi,
I am running the most recent version (as it installs by default), but there are multiple issues/ bugs that are coming up. Does anyone know how to run the previous version?  I tried doing an npm install, but the way that I tried to run it did not work.  Thanks for any pointers. 🙏

**Comment by SeriousBug** [comment]
Are you looking to install an older version of the extension, or the CLI?

Also, could you elaborate on what bugs you are seeing? We can prioritize fixing them.

**Reply by maiya-22** [reply]
Hi, thanks for quick reply!
Re the version: I installed dendron using VS code extension. Is that separate from the CLI?.  (Apologies, but I am new to the platform). I'm looking to install whichever one will get me to a stable previous release.
Re elaborating on the bugs: 1) One of the bugs was already reported and answered by someone else (the checkboxes). 2) Also, the graph sometimes loads and sometimes does not.  3) The css file is not updating the look of the graph. 4) The hierarchy is not refactoring.  It did not add a stub, and when I tried adding the page that I wanted to be added into the hierarchy, it still did not work.  

I am watching the onboarding session from Aug 27, 2020, and my setup is not showing the same behavior. I need to go through and do a detailed submission on each bug.  That's a little bit difficult while I am still learning the platform, so was thinking it would be more productive to switch to an earlier version, for now.

**Comment by SeriousBug** [comment]
The CLI and the extension are in fact different. In this case you are looking for the VSCode extension.

VSCode does have a way to install old versions of extensions: https://code.visualstudio.com/updates/v1_30#_install-previous-versions

But I'd recommend keeping the latest version if possible. Dendron changed a lot since Aug 2020, so it's normal that a lot of things don't match that video. Would you like to schedule an onboarding for yourself instead? Here's the link to schedule one: https://calendly.com/d/mqtk-rf7q/onboard?month=2022-01

**Reply by maiya-22** [reply]
I didn't know that was an option. I will sign up. Thanks!

**Comment by kevinslin** [comment]
To add on to what @SeriousBug is saying - we need to update that onboarding video. It uses a third party preview extension that has been deprecated. 

1. checkbox are not currently supported in the new preview
2. if you have specific repro steps, we can look into that
3. you will need to run reload window for the css to take into effect

hope that helps

**Reply by maiya-22** [reply]
When I tried to install an old version, I indeed got deprecation errors. I'll stay with the current version as advised.  I'm going to sign up for the onboarding, and once I am more comfortable I'll post any errors. Thanks again for your guys' help.
